src/nunit.analyzers/ValueSourceUsage/ValueSourceUsageAnalyzer.cs

```diff
@@ -62,14 +62,25 @@ public override void Initialize(AnalysisContext context)
         {
             context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
             context.EnableConcurrentExecution();
-            context.RegisterSyntaxNodeAction(x => AnalyzeAttribute(x), SyntaxKind.Attribute);
+            context.RegisterCompilationStartAction(AnalyzeCompilation);
         }
 
-        private static void AnalyzeAttribute(SyntaxNodeAnalysisContext context)
+        private static void AnalyzeCompilation(CompilationStartAnalysisContext context)
```

Comment:
Please rename to `AnalyzeCompilationStart`

Reply:
```suggestion
        private static void AnalyzeCompilationStart(CompilationStartAnalysisContext context)
```